src/components/settings/Settings.cpp

```diff
@@ -45,3 +47,14 @@ void Settings::SaveSettingsToFile() {
   fs.FileWrite(&settingsFile, reinterpret_cast<uint8_t*>(&settings), sizeof(settings));
   fs.FileClose(&settingsFile);
 }
+
+void Settings::uploadWakeUpMode() {
+  // Setup the motion sensor interrupt configuration
+  if(isWakeUpModeOn(Settings::WakeUpMode::DoubleTap)) {
+    acc.SetMotion(Drivers::MotionEvents::DoubleTap);
+  } else if(isWakeUpModeOn(Settings::WakeUpMode::SingleTap)) {
+    acc.SetMotion(Drivers::MotionEvents::SingleTap);
+  } else {
+    acc.SetMotion(Drivers::MotionEvents::None);
+  }
+}
```

Comment:
This seems to be generic wakeup refactoring, must be in a separate PR. Will break other unmerged ones touching similar parts of the code. Very hard to review in this context.

Reply:
This controls the gesture that the new accelerometer driver should use to wake up the watch. Useful if the touch sensor cannot generate these events.

Should I break out the accelerometer additions and refactoring (including the new SC7A20 driver) into a new PR?

Reply:
This change was postponed to a potential future PR.